arkit_app/DataView.swift

```diff
@@ -1,66 +1,71 @@
-        .browOuterUpLeft: 0,    .browOuterUpRight: 0,
-        
-        // Nose
-        .noseSneerLeft: 0,      .noseSneerRight: 0,
-        
-        // Cheek
-        .cheekPuff: 0,
-        .cheekSquintLeft: 0,    .cheekSquintRight:0,
-        
-        // Jaw
-        .jawForward: 0,
-        .jawOpen: 0,
-        .jawLeft: 0,            .jawRight: 0,
-        
-        // Mouth
-        .mouthClose: 0,
-        .mouthFunnel: 0,
-        .mouthPucker: 0,
-        .mouthLeft: 0,              .mouthRight: 0,
-        .mouthSmileLeft: 0,         .mouthSmileRight: 0,
-        .mouthFrownLeft: 0,         .mouthFrownRight: 0,
-        .mouthDimpleLeft: 0,        .mouthDimpleRight: 0,
-        .mouthStretchLeft: 0,       .mouthStretchRight: 0,
-        .mouthRollLower: 0,         .mouthRollUpper: 0,
-        .mouthShrugLower: 0,        .mouthShrugUpper: 0,
-        .mouthPressLeft: 0,         .mouthPressRight: 0,
-        .mouthLowerDownLeft: 0,     .mouthLowerDownRight: 0,
-        .mouthUpperUpLeft: 0,       .mouthUpperUpRight: 0,
-        
-        // Tongue
-        .tongueOut: 0,
-    ]
-    
     var arSessionManager = ARSessionManager();
+    @State private var blendShapes: [ARFaceAnchor.BlendShapeLocation: BlendShapeFeature] = Dictionary(
+        uniqueKeysWithValues: [
+            // Eye
+            .eyeBlinkLeft,       .eyeBlinkRight,
+            .eyeLookUpLeft,      .eyeLookUpRight,
+            .eyeLookDownLeft,    .eyeLookDownRight,
+            .eyeLookInLeft,      .eyeLookInRight,
+            .eyeLookOutLeft,     .eyeLookOutRight,
+            .eyeSquintLeft,      .eyeSquintRight,
+            .eyeWideLeft,        .eyeWideRight,
+            
+            // Brow
+            .browInnerUp,
+            .browDownLeft,       .browDownRight,
+            .browOuterUpLeft,    .browOuterUpRight,
+            
+            // Nose
+            .noseSneerLeft,      .noseSneerRight,
+            
+            // Cheek
+            .cheekPuff,
+            .cheekSquintLeft,    .cheekSquintRight,
+            
+            // Jaw
+            .jawForward,
+            .jawOpen,
+            .jawLeft,            .jawRight,
+            
+            // Mouth
+            .mouthClose,
+            .mouthFunnel,
+            .mouthPucker,
+            .mouthLeft,              .mouthRight,
+            .mouthSmileLeft,         .mouthSmileRight,
+            .mouthFrownLeft,         .mouthFrownRight,
+            .mouthDimpleLeft,        .mouthDimpleRight,
+            .mouthStretchLeft,       .mouthStretchRight,
+            .mouthRollLower,         .mouthRollUpper,
+            .mouthShrugLower,        .mouthShrugUpper,
+            .mouthPressLeft,         .mouthPressRight,
+            .mouthLowerDownLeft,     .mouthLowerDownRight,
+            .mouthUpperUpLeft,       .mouthUpperUpRight,
+            
+            // Tongue
+            .tongueOut,
+        ].map { (key: $0, value: BlendShapeFeature()) });
```

Comment:
This is insided the view - not sure if there's a runtime cost to this - surely it can't be THAT bad right... right?